homeassistant/components/hypontech/config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Config flow for the Hypontech Cloud integration."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from hyponcloud import AuthenticationError, HyponCloud
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> None:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+    session = async_get_clientsession(hass)
+    hypon = HyponCloud(data[CONF_USERNAME], data[CONF_PASSWORD], session)
+    try:
+        if not await hypon.connect():
+            raise CannotConnect
+    except TimeoutError:
+        raise CannotConnect from None
+    except AuthenticationError:
+        raise InvalidAuth from None
+
+
+class HypontechConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Hypontech Cloud."""
+
+    VERSION = 1
```

Comment:
No need as ConfigFlow version is always 1 by default

Reply:
Removed.